### DISCUSSION TITLE
NEW STAGE ISSUE

### DISCUSSION THREAD

**Discussion by CrazyCrabula** [KadeArchive/Kade-Engine#1789] [Q&A]
So I was creating a new stage, but every time I compile the game, the stage loads as the stage from week 1. Please help if you know what the issue is. Here is a video showing what's happening.

https://user-images.githubusercontent.com/88954406/129488572-6b9bfb7c-02df-4350-9193-513df5109e7e.mp4

**Comment by napstaa967** [comment]
1. this goes in discussions not issues
2. this happens when the stage cannot be loaded so it loads the default
make sure you've coded everything needed correcty for the stage to load

**Comment by CrazyCrabula** [comment]
Heres's a picture
![Screenshot (44)](https://user-images.githubusercontent.com/88954406/129495165-b4600d41-c22e-4b56-a12b-b43cb7b01574.png)
 of my code.

**Comment by napstaa967** [comment]
1. its not properly indented
2. missing ":" after 'chillwhitty'
